### ISSUE TITLE
SublimeLinter: generate_color_scheme: Preferences.sublime-settings invalid, aborting

### ISSUE THREAD

**Issue by aziz** [SublimeLinter/SublimeLinter3#180]
I'm seeing this error in my sublime console recently. 
checked my sublime `Preferences.sublime-settings` file and it's ok. I even validated the json using an online tool and it says it's all valid and good. 

Don't know why SublimeLinter is complaining about it. checked the source code and it probably cause by [`sublime.decode_value(json)`](https://github.com/SublimeLinter/SublimeLinter3/blob/8526e54505cab6abd248665393c7b3b31a962289/lint/util.py#L233). It might be caused by some UTF-8 characters I have in my `ignored_words` settings.

``` 
	"ignored_words":
	[
		"☐",
		"✔",
		"✘"
        ]
```

**Comment by skj3gg** [comment]
What linters are you using, what does your `Preferences.sublime-settings` look like, and what is your entire console output?

All this will help me piece together what's going wrong. `sublime.decode_value` is working fine for me with your unicode characters so there may be another pain point.

**Comment by aziz** [comment]
Here is my `Preferences.sublime-settings`
https://github.com/aziz/sublimeText3-Userfiles/blob/master/Preferences.sublime-settings

and here's a list of my linter plugins:
https://github.com/aziz/sublimeText3-Userfiles/blob/master/Package%20Control.sublime-settings#L72

- SublimeLinter
- SublimeLinter-coffeelint
- SublimeLinter-jshint
- SublimeLinter-json
- SublimeLinter-rubocop
- SublimeLinter-ruby
- SublimeLinter-shellcheck
